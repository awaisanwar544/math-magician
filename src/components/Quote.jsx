import React from 'react';

const Quote = () => (
  <div style={{ width: '100%', textAlign: 'center' }}>
    <h2>A quick brown jumped over the lazy dogs</h2>
  </div>
);

export default Quote;
