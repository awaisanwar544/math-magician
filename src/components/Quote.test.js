import React from 'react';
import renderer from 'react-test-renderer';

import Quote from './Quote';

describe('Calculator', () => {
  it('Displays correct layout of Calculator', () => {
    const tree = renderer
      .create(<Quote />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
