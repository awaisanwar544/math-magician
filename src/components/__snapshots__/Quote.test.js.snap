// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator Displays correct layout of Calculator 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
      "width": "100%",
    }
  }
>
  <h2>
    A quick brown jumped over the lazy dogs
  </h2>
</div>
`;
